src/stanchion_app.erl

```diff
@@ -52,3 +63,39 @@ start(_Type, _StartArgs) ->
 -spec stop(term()) -> ok.
 stop(_State) ->
     ok.
+
+check_admin_creds(Pid) ->
+    case application:get_env(stanchion, admin_key) of
+        {ok, "admin-key"} ->
+            lager:warning("admin.key is defined as default. Please create"
+                          " admin user and configure it.", []),
+            application:set_env(stanchion, admin_secret, "admin-secret");
+        {ok, KeyId} ->
+            case application:get_env(stanchion, admin_secret) of
+                {ok, _} ->
+                    lager:warning("admin.secret is ignored.");
+                _ ->
+                    ok
+            end,
+            StrongOpts = [{r, all}, {pr, one}, {notfound_ok, false}],
```

Comment:
When one of nodes which are responsible the admin user object, Stanchion will fail to start because the fallback vnode does not have it and responds `notfound`. Is `quorum` sufficient for R?